C/services/notification/data_availability_rule.cpp

```diff
@@ -32,20 +32,19 @@ static const char *default_config = QUOTE({
 		"displayName" : "Rule",
 		"readonly" : "true"
 	},
-	"condition" : {
-		"description" : "Audit log codes to consider",
-		"type" : "enumeration",
-		"options" : [ "ALL", "IN" ],
-		"default" : "ALL",
-		"displayName" : "Audit Log Code",
-		"order" : "1"
-	},
 	"auditCode" : {
-		"description" : "Comma separated list of audit log codes (e.g. CONCH,CONAD)",
+		"description" : "Audit log code to monitor, Leave blank if not required or set to * for all codes",
```

Comment:
@MarkRiddoch  Does this mean either no, one or all (*)? And two or more audit codes as X, Y cannot be monitored.

We must clarify in the description or add to documentation.